### PR TITLE
Adding Address Checker dev tool

### PR DESCRIPTION
**Pull request by gwendolyngoetz**
### Description

New dev tool to help test and debug address correction and service area validation in dev and staging environments. 

This is not accessible on production environments.

This was tossed together before we got much built up for thymeleaf. I'll have to come back later to transition it over.

Screenshots on the files tab.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Created unit and/or browser tests which fail without the change (if possible)

### CODE REVIEW

File: browser-test/src/dev/address_checker.test.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    page,
+  }) => {
+    await disableFeatureFlag(page, 'ESRI_ADDRESS_CORRECTION_ENABLED')
+    await disableFeatureFlag(
+      page,
+      'ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ENABLED',
+    )
+
+    await test.step('Navigate to address tools', async () => {
+      await page.click('#debug-content-modal-button')
```
Comment (dkatzz) [inline browser-test/src/dev/address_checker.test.ts:85]: **suggestion:** You could consider pulling this navigation out to a helper function

File: server/app/views/dev/AddressCheckerView.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import services.settings.SettingsManifest;
+import views.BaseHtmlLayout;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.JsBundle;
+import views.components.Icons;
+import views.dev.hx.CorrectAddressFormPartial;
+import views.dev.hx.ServiceAreaFormPartial;
+import views.style.BaseStyles;
+
```
Comment (dkatzz) [inline server/app/views/dev/AddressCheckerView.java:30]: **suggestion:** Could you a javadoc for this class and other new classes?

File: server/app/views/dev/DatabaseSeedView.java [hunk 3]
```diff
@@ -223,6 +224,17 @@ private SectionTag createEmailSection() {
...
   }
 
+  private SectionTag createAddressToolsSection() {
+    return section()
+        .with(h2("Address Tools").withClass("text-2xl"))
+        .with(p("View address lookup and eligibility results"))
+        .withClasses("flex", "flex-col", "gap-4", "border", "border-black", "p-4")
+        .with(
+            createLink(
+                "\uD83D\uDDFA\uFE0F Go to address tools",
```
Comment (dkatzz) [inline server/app/views/dev/DatabaseSeedView.java:234]: **question:** What is `\uD83D\uDDFA\uFE0F` for?

**Review by dkatzz — APPROVED** [review]
This is such a nice tool to have, thank you!!

File: server/app/views/dev/DatabaseSeedView.java (hunk 3 above)
Comment (gwendolyngoetz) [reply server/app/views/dev/DatabaseSeedView.java:234]: Map emoji 🗺️

;)

Comment (dkatzz) [reply server/app/views/dev/DatabaseSeedView.java]: Haha love it! Can you add a comment :)

Comment (gwendolyngoetz) [reply server/app/views/dev/DatabaseSeedView.java]: Already did

**Pull request merged**